.github/workflows/android-workflow.yaml

```diff
@@ -149,7 +149,35 @@ jobs:
         with:
           name: reports
           path: PennMobile/build/reports
+          
+  ktlint-check:
```

Comment:
do you want to make this part of our unit tests job? If not, should we make the unit tests job depend on this?

Reply:
I think it should be independent, it would be annoying if ur trying to check build/tests preliminarily and u can’t because of a lint